src/prefect/agent.py

```diff
@@ -496,7 +503,13 @@ async def _submit_run_and_capture_errors(
                 )
                 # Mark the task as started to prevent agent crash
                 task_status.started(exc)
-                await self._propose_failed_state(flow_run, exc)
+                await self._propose_crashed_state(
+                    flow_run,
+                    (
+                        "Flow run could not be submitted to infrastructure"
+                        f" {exception_traceback(exc)}."
+                    ),
+                )
```

Comment:
The traceback is logged by `self.logger.exception(...)` above, here we just want to include a short message about the error encountered.

Reply:
Removed the stack trace for the error excerpt here.